manual/source/Infrastructure/Use/FindSeqAnCMake.rst

```diff
@@ -77,26 +77,54 @@ First you should create a build directory, i.e. for cmake-builds everything happ
    # mkdir -p ~/devel/my_project-build/release
    # cd ~/devel/my_project-build/release
 
-By default, the ``cmake`` program will look for ``seqan-config.cmake`` in a predefined set of directories.
-Please read the documentation `find_project <https://cmake.org/cmake/help/v3.0/command/find_package.html>`_ to find out, which standard paths are searched.
-Depending on how you :ref:`installed SeqAn <infra-use-install>` it might be found by cmake automatically. 
-If not, you have to give the path containing the config file to cmake via the ``SeqAn_DIR`` argument on the command line, or you add the installation prefix of ``SeqAn`` via the ``CMAKE_PREFIX_PATH`` variable.
+Cmake supports two different modes to load settings from an external project: The **module** and the **config** mode. 
+Please read the `cmake documentation <https://cmake.org/cmake/help/v3.0/command/find_package.html>`_ to learn more about this feature.
 
-Also, CMake will look for the SeqAn include files in central locations such as ``/usr/local/include``. Again, depending on your installation this might *just work*. If not, you need to specify the location via the ``SEQAN_INCLUDE_PATH`` argument.
+Install SeqAn from package maintainer
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-When using operating system packages of SeqAn and the default compiler it might look like this:
+The recommended way for SeqAn 2.3 or newer is to use the config mode. 
+If you installed/updated SeqAn from one of the downstream package maintainer listed in :ref:`Getting Started with SeqAn <infra-use-install>`, then a file called ``seqan-config.cmake`` was installed in a system path that is automatically searched by the cmake system (see the cmake documentation for `find_package <https://cmake.org/cmake/help/v3.0/command/find_package.html>`_).
+If everything was done with default settings, than you can simply build your project like:
 
 .. code-block:: console
-
+   
    # cmake ../../my_project
+   
+In case you installed SeqAn into a specific <*prefix*> you can tell cmake to add this prefix to search for the ``seqan-config.cmake`` file.
+Assume you chose ``$HOME/local/seqan`` as the installation prefix, than your setup could look as the following:
+
+.. code-block:: console
+   
+   # cmake ../../my_project \
+      -DCMAKE_PREFIX_PATH="$HOME/local/seqan"
+
 
-If you instead did a full git checkout to your home-directory in the previous step, it might look like this:
+SeqAn from GitHub clone
+~~~~~~~~~~~~~~~~~~~~~~~
+
+In case you obtained SeqAn from a git clone, we still recommend to use the method described above and set the ``CMAKE_PREFIX_PATH`` to your SeqAn clone.
+But in addition you also have to specify the ``SEQAN_INCLUDE_PATH`` variable to find the SeqAn headers. 
+Assume you have cloned SeqAn into ``~/devel/seqan``, then your setup could look as the following:
```

Comment:
I don't understand why this is different from the paragraph above. Why do they not need to set the INCLUDE_PATH?